Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Incremental_Export_to_SIEM_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,30 @@
+Playbook to be run every 15 minutes via a job. Each run will get incremental updates for devices, alerts and vulnerabilities and send syslogs to the configured SIEM server.
```

Comment:
Gets incremental device, alert, and vulnerability updates from PANW IoT Cloud and send syslogs to the configured SIEM. This playbook should run every 15 minutes as a scheduled job.